src/memray/reporters/tree.py

```diff
@@ -75,41 +93,87 @@ async def update_text_area(self) -> None:
         if not self.__is_mounted or self.frame is None:
             return
 
-        _, file, line = self.frame.location
         text = self.query_one("#textarea", TextArea)
+
+        if self.frame.location is None:
+            text.clear()
+            return
+
+        _, file, line = self.frame.location
         delta = text.size.height // 2
         lines = linecache.getlines(file)[line - delta : line + delta]
 
         text.text = "\n".join(tuple(line.rstrip() for line in lines))
         text.select_line((delta - 1))
         text.show_line_numbers = False
 
+    def _get_content_by_label_id(self) -> Dict[str, str]:
```

Comment:
Can we confirm that we have a visual test that covers something where this text appears?

Reply:
Yep, we have tests exercising all 3 cases of this. There's one for the root node (which most of our pre-existing snapshot tests capture), one for a normal frame node (which a few of our pre-existing snasphot tests exercise), and one for the artificial excluded allocations row (which a newly added snapshot test exercises)